gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java

```diff
@@ -148,6 +147,10 @@ public GoogleMapsMapAdapter(
     map.setOnCameraMoveStartedListener(this::onCameraMoveStarted);
     map.setOnCameraMoveListener(this::onCameraMove);
     onCameraMove();
+
+    String url = "";
```

Comment:
We'll just pull this from the project—just hardcoded for testing the remote provider out for now.